go.mod

```diff
@@ -95,7 +95,7 @@ require (
 )
 
 require (
-	github.com/authgear/oauthrelyingparty v1.3.0
+	github.com/authgear/oauthrelyingparty v1.4.0
```

Comment:
Whenever we make changes to go.mod, we need to apply the same changes to custombuild/go.mod as well.

You may try `cd custombuild && go mod tidy`, but it should fail.
You can use `cd custombuild && GOPRIVATE='github.com/authgear/iamsmart' go mod tidy`

Reply:
In your `~/.config/git/config`, you can add

```
[url "ssh://git@github.com/authgear/iamsmart"]
	insteadOf = https://github.com/authgear/iamsmart
```